### DISCUSSION TITLE
BUILD FAILED Error

### DISCUSSION THREAD

**Discussion by Cemow** [pyenv/pyenv#2688] [General]
Hi,

I am trying to install python 3.6.8 with pyenv in order to work with it. The error is keep popping up. 
"BUILD FAILED (Ubuntu 22.04 using python-build 20180424)"

Does anybody know what might the issue be about this, Thank you.

**Comment by native-api** [comment]
This is a general error message when a build fails for any reason. A more specific error message should be elsewhere in the output or in the build log (the path to which is given in the output).

Find the real error message, search the project if it has been reported before, and if not, create a new issue and provide the necessary diagnostic information as per the issue template.

**Comment by Cemow** [comment]
@native-api 

Oh okay, here is a more detailed explanation:


BUILD FAILED (Ubuntu 22.04 using python-build 20180424)

Inspect or clean up the working tree at /tmp/python-build.20230425120208.5549
Results logged to /tmp/python-build.20230425120208.5549.log


Last 10 log lines:



if test "xupgrade" != "xno"  ; then \

	case upgrade in \

		upgrade) ensurepip="--upgrade" ;; \

		install|*) ensurepip="" ;; \

	esac; \

	LD_LIBRARY_PATH=/tmp/python-build.20230425120208.5549/Python-3.6.8 ./python -E -m ensurepip \

		$ensurepip --root=/ ; \

fi

Segmentation fault (core dumped)

make: *** [Makefile:1102: install] Error 139

This is what is happening: I tried to install multiple additional libraries but couldnt figure out the problem.

**Comment by Cemow** [comment]
@native-api 

I tried to install 3.6.15 instead of 3.6.8 and it worked. There is an issue with 3.6.8, Thanks anyway.
